bankaccount.go

```diff
@@ -155,7 +166,7 @@ type BankAccount struct {
 	Account                *Account                           `json:"account"`
 	AccountHolderName      string                             `json:"account_holder_name"`
 	AccountHolderType      BankAccountAccountHolderType       `json:"account_holder_type"`
-	AccountType            *string                            `form:"account_type"`
+	AccountType            string                             `json:"account_type"`
```

Comment:
I think this is right?

Reply:
Good catch - I probably made a mistake manually porting this in https://github.com/stripe/stripe-go/pull/1322/files#diff-ccd7af701903ec5371bec49de5a76c5574d82e613696da9a0046258b07e476d1R158.